homework-submission/routing/src/components/ApiCall/components/Users.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+
+// Users
+class Users extends Component {
+  state = {
+    people: [],
+  };
+
+  componentDidMount() {
+    fetch('https://uinames.com/api/?amount=10')
+      .then(res => res.json())
+      .then(users =>
+        this.setState({
+          isLoaded: true,
+          people: users,
+        }),
+      )
+      .catch(err => err);
+  }
+
+  render() {
+    const HeaderComponent = ({ content }) => <h1>{content}</h1>;
+
+    const ListComponent = ({ title, content }) => (
+      <li>
+        {title}: {content}
+      </li>
+    );
+
+    return (
+      <div>
+        <ul>
+          {this.state.people.map((user, i) => (
+            <div>
```

Comment:
If you map over items in an array you need the set a `key`, but the key should be on the first element inside the map function; in this case the `<div>` element, setting the key on the components inside the div still gives you errors. Try adding the key to the div element instead of the components.